ViewModel/StoreSwitchModel.php

```diff
@@ -183,18 +206,18 @@ public function getStoreCountyCode(StoreInterface $store): string
 
     /**
      * Get the formatted label for the dropdown, based on the format configuration.
-     * 
+     *
      * @param StoreInterface $store
      *
      * @return string
      * @throws \Exception
      */
-    public function getStoreSwitchLabel(StoreInterface $store): string 
+    public function getStoreSwitchLabel(StoreInterface $store): string
     {
         $showCountryOnly = $this->scopeConfig->getValue(self::MODULE_SHOW_COUNTRY_ONLY_CONFIG_PATH, ScopeInterface::SCOPE_STORE, $store->getId());
         if($showCountryOnly) {
             return $this->getStoreCountyCode($store);
         }
         return $this->getParsedLanguage($store).'&nbsp;('.$this->getStoreCountyCode($store).')';
     }
-}
\ No newline at end of file
+}
```

Comment:
Do you plan to add a config.xml for configurablity via admin panel?

Reply:
No. Should I add one?

Reply:
I don't think you'd add websites that often or change their configuration that an admin panel configuration would make sense.

Reply:
yeah, but things seem uncomplete if not configurable via admin panel ... I would suggest to add it, it's not hard :-)

Reply:
Added :upside_down_face: